packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/SelectIModel.tsx

```diff
@@ -50,7 +50,7 @@ const useProgressIndicator = (iModel: IModelFull) => {
         );
       });
     }
-  }, [userSettings]);
+  }, [userSettings, iModel.id, iModel.projectId]);
```

Comment:
were the changes in these files also made in the desktop test app here?

Reply:
yes, the changes came from there originally